packages/gatsby/src/commands/develop.js

```diff
@@ -123,9 +123,14 @@ async function startServer(program) {
 
   app.use(cors())
 
+  /**
+   * Pattern matching all endpoints with graphql or graphiql with 1 or more leading underscores
+   */
+  const graphqlEndpoint = `/_+graphi{0,1}ql`
```

Comment:
swap `{0,1}` into `?`. I like this syntax more but I might be the only one :p (so if you don't agree, please ignore)
```suggestion
  const graphqlEndpoint = `/_+graphi?ql`
```

Reply:
Good point, I didn't think about using an optional operator instead of a range.